app/services/allocations_exporter.rb

```diff
@@ -60,6 +66,12 @@ def render(csv, query)
         (school.in_virtual_cap_pool? ? school.std_device_allocation&.allocation : nil),
         (school.in_virtual_cap_pool? ? school.std_device_allocation&.cap : nil),
         (school.in_virtual_cap_pool? ? school.std_device_allocation&.devices_ordered : nil),
+        school.coms_device_allocation&.raw_allocation,
```

Comment:
might also want add `:coms_device_allocation` to the `.includes(...)` of the `schools` query, to avoid n+1 queries

Reply:
good spot, i'll add that in now